src/modules/commander/HomePosition.cpp

```diff
@@ -313,6 +305,18 @@ void HomePosition::update(bool set_automatically, bool check_if_changed)
 {
 	_local_position_sub.update();
 	_global_position_sub.update();
+	_vehicle_gps_position_sub.update();
+
+	if (_vehicle_gps_position_sub.updated()) {
```

Comment:
@bkueng is `.updated() `on `SubscriptionData` working the same as on normal Subscriptions?

Reply:
Yes but it's better to store only the few fields that you need.

Reply:
reverted https://github.com/PX4/PX4-Autopilot/commit/72260d00866d0aa39aa20ee434e947b13bfc2201